templates/Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,81 @@
+// Run every 5 minutes 
+CRON_SETTINGS = '''*/5 * * * *'''
+
+pipeline {
+    agent any
+
+    triggers {
+        cron(CRON_SETTINGS)
+    }
+
+    environment {
+        // [Mandatory only for project which npm, pip, pipenv, nuget and dotnet]
```

Comment:
[Mandatory only for project which npm, pip, pipenv, nuget and dotnet]
-->
[Mandatory only for projects which use npm, pip, pipenv, nuget and dotnet to download their dependencies]

This same error exists on the GitLab section in the README, so let's fix it there as well.